src/components/EntryForm.js

```diff
@@ -0,0 +1,31 @@
+//import {useState} from 'react';
+
+import useVacationStore from './hooks/useVacationStore';
+// import Vacations from './vacations';
+
+export default function EntryForm() {
+	// 	const [entries, setEntries] = useState('');
+
+	return (
+		<>
+			<form>
+				<label htmlFor="chooseVacation">Choose a vacation: </label>
+				<select name="vacations" id="vacation-select">
```

Comment:
The `htmlFor` attribute of your label has a different value than the `id` attribute of your `select` element. You should be consistent otherwise the connection between form element and label won't work.